JSDemos/Demos/List/ListWithSearchBar/description.md

```diff
@@ -1 +1,7 @@
-This demo shows the **List** UI component with the searching capability enabled. Depending on&nbsp;the search mode, the UI component finds entries that contain your search string or&nbsp;entries that only start with&nbsp;it.
\ No newline at end of file
+To add a search bar to the **List** and enable the search functionality, do the following:
+
+- Set the [searchEnabled](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchEnabled) property to **true**.   
+
+- Specify the [searchExpr](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchMode) property. It is used to compare the search string against a specific field in your data objects. In this example, the search string is compared against the *"Name"* field.
+
+- Use the [searchMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchMode) property to specify whether found list items should contain, be equal to, or start with the search string. In this example, you can use the drop-down menu under the List to change the **searchMode**.   
```

Comment:
Minor thing -  I wouldn't use "found list items". I'd also change "be equal to" to "match".

```suggestion
- Use the [searchMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchMode) property to specify whether list items should start with, contain, or match the search string. In this example, you can use the drop-down menu under the List to change the **searchMode**.   
```